ui/hooks/gasFeeInput/useMaxPriorityFeePerGasInput.js

```diff
@@ -34,13 +36,18 @@ const getMaxPriorityFeePerGasFromTransaction = (transaction) => {
  *  method to update the maxPriorityFeePerGas.
  */
 export function useMaxPriorityFeePerGasInput({
-  supportsEIP1559V2,
+  editGasMode,
   estimateToUse,
   gasEstimateType,
   gasFeeEstimates,
   gasLimit,
+  supportsEIP1559V2,
   transaction,
 }) {
+  const swapCustomMaxPriorityFeePerGas = useSelector(
```

Comment:
could we add a comment here explaining why we use the `getCustomMaxPriorityFeePerGas` for this value?

Reply:
I found that this was redundant, I just removed.